source/PlayerInfo.cpp

```diff
@@ -668,21 +669,8 @@ Account &PlayerInfo::Accounts()
 // Calculate how much the player pays in daily salaries.
 int64_t PlayerInfo::Salaries() const
 {
-	// Don't count extra crew on anything but the flagship.
-	int64_t crew = 0;
-	const Ship *flagship = Flagship();
-	if(flagship)
-		crew = flagship->Crew() - flagship->RequiredCrew();
-	
-	// A ship that is "parked" remains on a planet and requires no salaries.
-	for(const shared_ptr<Ship> &ship : ships)
-		if(!ship->IsParked() && !ship->IsDestroyed())
-			crew += ship->RequiredCrew();
-	if(!crew)
-		return 0;
-	
-	// Every crew member except the player receives 100 credits per day.
-	return 100 * (crew - 1);
+	const Ship * playerFlagship = Flagship(); 
+	return Crew::CalculateSalaries(ships, playerFlagship);
```

Comment:
Just use `Flagship()` here

Reply:
We can't because the compiler complains that the argument has to be an `lvalue`.

Reply:
That's because your function is still passing it as a reference instead of by value.